packages/typescript-estree/src/create-program/createProjectService.ts

```diff
@@ -94,6 +94,12 @@ export function createProjectService(
       );
     }
 
+    service.setHostConfiguration({
+      preferences: {
+        includePackageJsonAutoImports: 'off',
+      },
+    });
```

Comment:
@jakebailey Is this intentionally inside the `if (options.defaultProject) {`?

Reply:
Nope...

Reply:
2 space indent is way too small for me to notice :(((